battlegear mod src/minecraft/mods/battlegear2/coremod/transformers/TransformerBase.java

```diff
@@ -41,13 +41,14 @@ public TransformerBase(String classPath){
             boolean success = processFields(cn.fields) && processMethods(cn.methods);
             addInterface(cn.interfaces);
             
-			ClassWriter cw = new ClassWriter(0);
+			ClassWriter cw = new ClassWriter(ClassWriter.COMPUTE_MAXS);
```

Comment:
As I recall one of the ASM tweaks didn't work with the code compiled by Eclipse (only code compiled by javac) and this was the easiest fix.